<?xml version="1.0"?>
<launch>

	<arg name="robot" />

	<rosparam file="$(find cob_controller_configuration_gazebo)/ros/controller/arm_controller_$(arg robot).yaml" command="load"/>

	<node name="arm_controller_spawner_left" pkg="cob_controller_configuration_gazebo" type="spawner" args="arm_left_controller" />
	<node name="arm_controller_spawner_right" pkg="cob_controller_configuration_gazebo" type="spawner" args="arm_right_controller" />
	
	
	<group ns="arm_left_controller">
		<node name="arm_left_joint_trajectory_action_node" pkg="joint_trajectory_action" type="joint_trajectory_action" />
	</group>
	<group ns="arm_right_controller">
		<node name="arm_right_joint_trajectory_action_node" pkg="joint_trajectory_action" type="joint_trajectory_action" />
	</group>

</launch>
